cmd/minishift/cmd/start_preflight.go

```diff
@@ -46,6 +47,11 @@ const (
 func preflightChecksBeforeStartingHost() {
 	driverErrorMessage := "See the 'Setting Up the Driver Plug-in' topic for more information"
 
+	err := validations.IsValidDriver("", viper.GetString(configCmd.VmDriver.Name))
```

Comment:
the `name` parameter does not have to be empty...
is it possible still wrap this in a preflightCheck function? As this will then be consistent with execution and reporting.